x/wasm/keeper/proposal_integration_test.go

```diff
@@ -111,8 +111,12 @@ func TestInstantiateProposal(t *testing.T) {
 	}}
 	assert.Equal(t, expHistory, wasmKeeper.GetContractHistory(ctx, contractAddr))
 	// and event
-	require.Len(t, em.Events(), 2, "%#v", em.Events())
-	require.Len(t, em.Events()[1].Attributes, 4)
+	require.Len(t, em.Events(), 3, "%#v", em.Events())
+	require.Equal(t, types.WasmModuleEventType, em.Events()[0].Type)
```

Comment:
This is the `type: wasm` event?

Again, I figure this should be after instantiate